test/integrations/Elastic.Apm.AspNetCore.Tests/BodyCapturingTests.cs

```diff
@@ -285,6 +284,206 @@ public async Task ApmMiddleware_ShouldSkipCapturing_WhenInvalidContentType()
 			result.StatusCode.Should().Be(HttpStatusCode.UnsupportedMediaType);
 		}
 
+		[Fact]
+		public async Task When_CaptureBodyConfigurationToAllAndSuccessfulCall_Should_CaptureBody()
+		{
+			var sutEnv = StartSutEnv(CreateConfiguration("all"));
```

Comment:
nit: For` the "all" scenario, you can call `CreateConfiguration()` as it defaults to all.